examples/MvcWeb/Controllers/ProjectController.cs

```diff
@@ -1,26 +1,49 @@
-﻿using System.Threading.Tasks;
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
-using MvcWeb.Models;
-using MvcWeb.Services;
+using MvcWeb.TheraLang.Entities;
+using MvcWeb.TheraLang.Repository;
 
 namespace MvcWeb.Controllers
 {
-    [Route("/api/projects/")]
+    [Route("api/project")]
+    [ApiController]
+
     public class ProjectController : ControllerBase
     {
         public ProjectController(IProjectService projectService)
         {
-            ProjectService = projectService;
+                _projectService = projectService;
         }
+            private readonly IProjectService _projectService;
 
-        private IProjectService ProjectService { get; }
+            public IEnumerable<Project> GetAllProjects()
+            {
+                return _projectService.GetAll();
+            }
 
-        [HttpPost("create")]
-        public async Task<ActionResult<bool>> CreateProject(ProjectViewModel projectViewModel)
-        {
-            var ms = ModelState;
-            if (!ModelState.IsValid) return false;
-            return await ProjectService.TryAddProject(projectViewModel);
+            [HttpPut("{id}")]
+            public async Task<IActionResult> ApproveStatusId(int id)
```

Comment:
just approve, because you do approve of project - not status
```suggestion
            public async Task<IActionResult> Approve(int id)
```